doc/specifications/rest/REST-API-V2.rst

```diff
@@ -7076,6 +7139,37 @@ ContentCreate XML Schema
       <xsd:element name="ContentCreate" type="contentCreateType"></xsd:element>
     </xsd:schema>
 
+
+.. _SimplifiedContentCreate:
+
+Simplified ContentCreate Example
+--------------------------------
+
+.. code:: json
+
+    {
+        "ContentCreate": {
+            "ContentType": "/content/types/_by_identified/article",
+            "LocationCreate": {
+                "_parentUrlAliasPath": "/content/urlaliases/_by_path/places-tastes/tastes/"
+            },
+            "ContentSection": "/content/section/_by_identifier/places",
+            "Owner": "/user/users/_by_login/jessica",
+            "alwaysAvailable": "false",
+            "modificationDate": "2017-03-02T12:00:00",
+            "remoteId": "remote123456",
+            "fields": {
+                "en-US": {
+                    "title": "This is a title",
+                    "intro": {
+                        "xhtml5edit": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<section xmlns=\"http://ez.no/namespaces/ezpublish5/xhtml5/edit\"><p>Article intro.</p></section>\n",
+                        "xml": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<section xmlns=\"http://docbook.org/ns/docbook\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:ezxhtml=\"http://ez.no/xmlns/ezpublish/docbook/xhtml\" xmlns:ezcustom=\"http://ez.no/xmlns/ezpublish/docbook/custom\" version=\"5.0-variant ezpublish-1.0\"><para>Article intro.</para></section>\n"
```

Comment:
@alongosz The latter one. The attributes set for a field are only determined by the field type.